static/app/components/replays/replayView.tsx

```diff
@@ -10,54 +9,48 @@ import ReplayPlayer from 'sentry/components/replays/replayPlayer';
 import space from 'sentry/styles/space';
 
 type Props = {
-  isFullscreen: boolean;
   toggleFullscreen: () => void;
 };
 
-function ReplayView({isFullscreen, toggleFullscreen}: Props) {
+function ReplayView({toggleFullscreen}: Props) {
   return (
     <Fragment>
       <ReplayCurrentUrl />
-      <PanelNoMargin isFullscreen={isFullscreen}>
-        <PanelHeader disablePadding noBorder>
+      <PlayerContainer>
+        <Panel>
           <ReplayPlayer />
-        </PanelHeader>
+        </Panel>
         <ScrubberMouseTracking>
           <PlayerScrubber />
         </ScrubberMouseTracking>
-      </PanelNoMargin>
-      <ReplayControllerWrapper>
-        <ReplayController toggleFullscreen={toggleFullscreen} />
-      </ReplayControllerWrapper>
+      </PlayerContainer>
+      <ReplayController toggleFullscreen={toggleFullscreen} />
     </Fragment>
   );
 }
 
-const ReplayControllerWrapper = styled(PanelBody)`
-  padding-top: ${space(1)};
-`;
+const PlayerContainer = styled('div')`
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  flex-grow: 1;
+  overflow: hidden;
 
-const PanelNoMargin = styled(Panel)<{isFullscreen: boolean}>`
-  margin-top: ${space(1)};
-  margin-bottom: 0;
-  height: 100%;
-  display: grid;
-  grid-template-rows: 1fr auto;
+  padding-bottom: ${space(1)};
+  margin-bottom: -${space(1)};
```

Comment:
These are here because of the first issue i mentioned in the desc.

The scrubber nub was being cutoff on the bottom by overflow:hidden. So we fudge some space. css is awesome!